### ISSUE TITLE
Extraction error

### ISSUE THREAD

**Issue by Devyatyi9** [donmai-me/WannaCRI#8]
I've got some errors when a was trying extract usm file and even use command 'probeusm'. File has been created via Scaleform VideoEncoder (CRI Medianoche [NR] Ver.1.70.00 - 2013-03-08) with subtitles. And video correctly works in game.
I didn't find any mention of the `@SBT` tag in the log file.
[test_1_usm.usm_NSi.log](https://github.com/donmai-me/WannaCRI/files/9335490/test_1_usm.usm_NSi.log)
[test_1_usm.zip](https://github.com/donmai-me/WannaCRI/files/9335497/test_1_usm.zip)
[screenshot 1](https://user-images.githubusercontent.com/5161472/184588748-9a72d809-63d7-492b-86ec-f2e387d3f08c.png)
[screenshot 2](https://user-images.githubusercontent.com/5161472/184588834-4bab3eb7-32d8-4c34-9801-58392572317b.png)

**Comment by Devyatyi9** [comment]
Also checked file from Honkai Impact 3rd, same error.
[1.7_CG01_mux.usm_KYe.log.zip](https://github.com/donmai-me/WannaCRI/files/9335601/1.7_CG01_mux.usm_KYe.log.zip)

**Comment by LittleChungi** [comment]
WannaCri is missing some tags related to USM chunks, and overall the extraction method can be made better. Since the header basically includes all type chunks included in the USM within the CRID part.

I checked your profile and as you know, the SBT format is not particularly specific, the language ID's can be arbitrary, and won't always match with Honkai labeling. I would assume parsing those back to SBT format is quite easy after extraction to SRT, from datetime -> milliseconds basically.

As for extracting those if you want, you can use my [tool](https://github.com/LittleChungi/PyCriCodecs) which will extract the SBT tag, as well as convert it to SRT for all available languages, although I couldn't find a way to decide what the language ID specifically is, as it's not consistent and the value is most likely arbitrary. However it's not as easy as WannaCri's, you would need some python experience.

Although an improvement for if they still want to hardcode the Tag types, is to add all of those:
```python
class USMChunckHeaderType(Enum):
    CRID  = b"CRID" # Header.
    SFSH  = b"SFSH" # SofDec1 Header?
    SFV   = b"@SFV" # Video (VP9/H264/MPEG).
    SFA   = b"@SFA" # Audio (HCA/ADX).
    ALP   = b"@ALP" # Rare. (Alpha video information.)
    CUE   = b"@CUE" # Rare. (Unknown.)
    SBT   = b"@SBT" # Rare. (Subtitle information.)
    AHX   = b"@AHX" # Rare. (Ahx audio file? Used for SofDec1 only?)
    USR   = b"@USR" # Rare. (User data?)
    PST   = b"@PST" # Rare. (Unknown.)
```
Although as I said, probably better to reserve this for building, for extraction, the CRID part says it all.

Also for `@UTF` parsing, I couldn't help but notice that the indexes might be wrong in WannaCri, there's a conflicting usage for those across tools, but I did found using the even index for unsigned type values is correct as otherwise it would give us some incorrect values in some CPK's/ACB's. Although, I am not sure myself.

**Comment by Devyatyi9** [comment]
@LittleChungi Thanks!
> what the language ID specifically is
It's subtitle track.
Earlier, before I found out about this program, I decided to call it lang id.
![2022-08-18_08-26-03](https://user-images.githubusercontent.com/5161472/185293628-e21bcf91-4f71-4eb1-af2c-1630ec5f4d5b.png)

**Comment by LittleChungi** [comment]
Yeah I saw this in the SDK, the numbers are arbitrary sadly, so what is specified for a certain game is not universal. Glad I saw an example of this chunk here in use though, helped me write the SBT -> SRT converter for my lib.

**Comment by donmai-me** [comment]
Thanks for taking the time in reporting this and also thank you to LittleChungi for the helpful information. I don't think I can currently add support for subtitles and all the other types of USM chunks, mostly because I lack the time and information needed. However, if you're only interested in extracting the video and audio data then I can make the other USM chunks act like stubs in the meantime.
